containers/kubernetes-pod.json

```diff
@@ -6,30 +6,34 @@
          "kind":"ReplicationController",
          "apiVersion":"v1",
          "metadata":{
-            "name":"cockpit-kube",
+            "name":"kubernetes-cockpit",
             "labels":{
-               "name":"cockpit-kube"
+               "name":"kubernetes-cockpit"
             }
          },
          "spec":{
             "replicas":1,
             "selector":{
-               "name":"cockpit-kube"
+               "name":"kubernetes-cockpit"
             },
             "template":{
                "metadata":{
                   "labels":{
-                     "name":"cockpit-kube"
+                     "name":"kubernetes-cockpit"
                   }
                },
                "spec":{
                   "containers":[{
-                    "name": "cockpit-kube",
-                    "image": "cockpit/kubernetes",
+                    "name": "kubernetes-cockpit",
+                    "image": "cockpit/kubernetes:latest",
                     "ports":[{
                         "containerPort":9090,
                         "protocol":"TCP"
-                     }]
+                     }],
+                    "env":[{
+                        "name": "KUBE_INSECURE",
```

Comment:
I've put it in this file so people know where to edit it and change the setting.